src/main/java/com/jc/tm/database/dao/TaskDaoImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.jc.tm.database.dao;
+
+import com.jc.tm.database.entity.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskDaoImpl implements TaskDao{
+
+    List<Task> tasks = new ArrayList<>();
```

Comment:
dao class usually is stateless object, and tasks field not necessary, pls remove it

Reply:
It is my second commit. In third it deleted already